### ISSUE TITLE
Too many arguments for function: jsonpath#echo

### ISSUE THREAD

**Issue by jsolbrig** [mogelbrod/vim-jsonpath#12]
## Dislaimer

I just installed the plugin, so it is very likely that I'm doing something wrong in my configuration.

## Problem

When I execute `:JsonPath` I get the following:
```
Error dertected while processing function jsonpath#command:
line    2:
E118: Too many arguments for function: jsonpath#echo
```

## Vim version

I'm using VIM version 8.2.754 *without* Python 3 support.

## My Setup

To test this, I removed just about everything from my `vimrc`, deleted all of my previously installed bundles, then installed only JsonPath. My minimal `vimrc` looks like this:

```
set nocompatible
" set termguicolors
filetype off

" Set the runtime path to include Vundle and initialize
set rtp+=~/.vim/bundle/Vundle.vim
call vundle#begin()

" let Vundle manage Vundle
Plugin 'VundleVim/Vundle.vim'
"
" json Path information
Plugin 'mogelbrod/vim-jsonpath'
au FileType json noremap <buffer> <silent> <expr> <leader>p jsonpath#echo()
au FileType json noremap <buffer> <silent> <expr> <leader>g jsonpath#goto()

" All of your plugins must be added before this
call vundle#end()
filetype plugin indent on
```

**Comment by mogelbrod** [comment]
Thanks for the bug report! I've released a fix as part of [v0.5.0](https://github.com/mogelbrod/vim-jsonpath/releases/tag/v0.5.0).

**Comment by jsolbrig** [comment]
Thanks for the bug fix! This works for me now.
